marshal_test.go

```diff
@@ -1217,6 +1231,46 @@ func TestMarshalPointer(t *testing.T) {
 	}
 }
 
+func TestMarshalTime(t *testing.T) {
+	durationS := "1h10m10s"
+	duration, _ := time.ParseDuration(durationS)
+	expectedData := encBigInt(duration.Nanoseconds())
+	var marshalTimeTests = []struct {
+		Info  TypeInfo
+		Data  []byte
+		Value interface{}
+	}{
+		{
+			NativeType{proto: 5, typ: TypeTime},
```

Comment:
The `time` and `date` types were added in CQL protocol v4.

Reply:
Fixing!